gc/base/HeapMap.hpp

```diff
@@ -255,6 +255,18 @@ class MM_HeapMap : public MM_BaseVirtual
 		return false;
 	}
 	
+	MMINLINE uintptr_t*
+	getSlotPtr(uintptr_t index) const
+	{
+		return &(getHeapMapBits()[index]);
+	}
+
+	MMINLINE uintptr_t*
+	getSlotPtrForAddress(omrobjectptr_t address) const
+	{
+		return getSlotPtr(getSlotIndex(address));
+	}
```

Comment:
Same here, this function can be overloaded, so the const version returns a const pointer:
```suggestion
	MMINLINE uintptr_t *
	getSlotPtrForAddress(omrobjectptr_t address)
	{
		return getSlotPtr(getSlotIndex(address));
	}

	MMINLINE const uintptr_t *
	getSlotPtrForAddress(omrobjectptr_t address) const
	{
		return getSlotPtr(getSlotIndex(address));
	}
```